devfiles/quarkus-commandmode/devfile.yaml

```
@@ -0,0 +1,100 @@
+---
+apiVersion: 1.0.0
+metadata:
+  generateName: quarkus-commandemode-
```

commande is looking too frenchy :-)
```suggestion
  generateName: quarkus-commandmode-
```